packages/stream/src/combinators/state.ts

```diff
@@ -0,0 +1,53 @@
+import { Stream } from '@motorcycle/types'
+import { curry3 } from '167'
+import { scan } from './scan'
+import { switchMap } from './switchMap'
+
+/**
+ * Especially useful when keeping local state that also needs to be updated
+ * from an outside source.
+ * @name state<A, B>(f: (acc: A, value: B) => A, seed$: Stream<A>, values$: Stream<B>): Stream<A>
+ * @example 
+ * import { Stream } from '@motorcycle/types'
+ * import { query, dragOverEvent, dragstartEvent, dropEvent } from '@motorcycle/dom'
```

Comment:
If we're changing `dragstartEvent` to correct camelCase, we may as well update it here, too.